Vagrantfile

```diff
@@ -12,7 +12,7 @@ Vagrant.configure("2") do |config|
   config.vm.hostname = "mrover-devbox"
 
   # Map all ports 8000-8020 for development work
-  (8000...8020).each do |p| 
+  (8000...8021).each do |p| 
```

Comment:
Is there a reason you're changing this?

Reply:
I think these stray commits are because his branch is out of date with respect to `umrover:master`. A `git pull --rebase upstream master` should fix these issues, when i did it when testing his code with my travis updates those extra commits disappeared.